### ISSUE TITLE
Knowledge Base "unfriendly"

### ISSUE THREAD

**Issue by Ttech** [freenode/web-7.0#194]

**Comment by svbeon** [comment]
In what way? What would be better? Why?

**Comment by mistohise** [comment]
This was entirely deliberate. The previous concept of having a massive list of stuff - one which, essentially, grew dynamically (and, thus messily) over time, left us with an awful globulous mess of stuff for people to wade through.

**Comment by Fuchs** [comment]
I agree with most of it, however, I don't think that one huge site is the solution. 

If things are further split up and it grows, then the overview (/kb/all) should be a good place to find stuff. However, as it only contains a title and, maybe, teaser, it should end up less messy than the old site. So we shall see how that works out.

I opened various tickets for the various things you suggest, e.g. there is now a page search which should help, especially when things are being more split up as planned, and the titles should be fixed mostly by now, I assume the rest will be shortly.
